tensorlayer/dataflow/dataset/cifar10.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import os
+import pickle
+import sys
+import numpy as np
+
+from ..base import Dataset
+from ..utils import maybe_download_and_extract
+
+CIFAR10_URL = 'https://www.cs.toronto.edu/~kriz/cifar-10-python.tar.gz'
+
+
+class CIFAR10(Dataset):
```

Comment:
move to tl.files?? any idea?

Reply:
I think those data-related functions originally in tl.files should be moved to tl.data in the future so that all data-related functionalities are in the same place.